packages/shared/src/components/utilities.tsx

```diff
@@ -77,6 +77,9 @@ export const getThemeColor = (color: string, fallback: string): ThemeColor => {
   return themeColors[color] ?? themeColors[fallback];
 };
 
+export const commentEventName = (upvoted: boolean): string =>
```

Comment:
Perhaps it's better?

```suggestion
export const upvoteCommentEventName = (upvoted: boolean): string =>
```